### ISSUE TITLE
JAWS 2021 is not reading aria-describedby elements contents when attribute is added to role="dialog"

### ISSUE THREAD

**Issue by markmagennis** [FreedomScientific/VFO-standards-support#541]
### Summary

JAWS 2021 is not reading aria-describedby element contents when attribute is added to role="dialog".

N.B. For convenience I have copied this from a previous issue https://github.com/FreedomScientific/VFO-standards-support/issues/3 which described this problem occurring in JAWS 18. This was fixed in 2019 and 2020 but seems to have reappeared in 2021. The following description refers to the same codepen for testing (which is not therefore my own) and gives slightly different instructions that demonstrate the issue more clearly.

1. Go to https://codepen.io/lifter035/project/full/AxKWap/
2. Activate the 'Open Dialog Focus on Close' button

### Expected result

JAWS should announce dialog aria-labelledby and aria-describedby elements contents when dialog is shown

### Actual result

JAWS does not announce the aria-describedby elements contents, only the aria-labelledby

#### JAWS version and build number

2021.2105.53 ILM
 
#### Operating System and version

Windows 10 Enterprise version 1803

#### Browser and version:

Google Chrome Version 91.0.4472.77 (Official Build) (64-bit)

**Comment by JAWS-test** [comment]
Can confirm the problem. It also occurs on the page https://www.w3.org/TR/wai-aria-practices-1.1/examples/dialog-modal/dialog.html

**Comment by greypants** [comment]
I'm having the opposite problem - I'm noticing Jaws *only* reading `aria-describedby` and ignoring `aria-labelledby`.

It's reproducible on the examples linked to here: https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html

Specifically this one: 
<img width="1216" alt="image" src="https://user-images.githubusercontent.com/647549/163593827-f7158895-40ef-480d-b834-e64c329700a7.png">

When I open it, it reads the description 3x and _does not_ read the title.

If I remove the `aria-labelledby`, it reads the whole dialog contents once.

**Comment by JAWS-test** [comment]
@greypants Strange that you have a different screen reader output. In my test environment it is still the case that with JAWS 2022 and Edge only the label is output and not the description
